src/main/kotlin/org/jitsi/nlj/stats/EndpointConnectionStats.kt

```diff
@@ -87,7 +94,7 @@ class EndpointConnectionStats(
     private fun processReportBlock(receivedTime: Long, reportBlock: RtcpReportBlock) {
         if (reportBlock.lastSrTimestamp > 0 && reportBlock.delaySinceLastSr > 0) {
             // We need to know when we sent the last SR
-            val srSentTime = srSentTimes.getOrDefault(reportBlock.lastSrTimestamp, -1)
+            val srSentTime = srSentTimes.getOrDefault(Pair(reportBlock.ssrc, reportBlock.lastSrTimestamp), -1)
```

Comment:
i wonder, is it worth considering tracking this data as `Map<Ssrc, Map<Timestamp, SentTime>>`?  it'd a void the need to create a pair on each lookup--maybe not worth worrying about at all, just throwing it out there.

...this also makes me think we should have `typealias`es for `Ssrc`and `Timestamp` as well (at least until maybe we can move to inline classes).  but, a change for another time.

Reply:
The problem with the nested map is that it makes the LRUCache a lot more complicated -- you end up with double LRUs, which can potentially lead to things lingering a lot longer.

Agreed about the `typealias`, both that it's a good idea and that it's for later.